### PR TITLE
[huggingface_pytorch] Add labels to dockerfile

### PR DESCRIPTION
**Pull request by xyang16**
*Issue #, if available:*

*Description of changes:* Add labels to dockerfile

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: huggingface/pytorch/tgi/docker/0.5.0/py3/cu118/Dockerfile.gpu [hunk 1]
```diff
@@ -129,3 +129,8 @@ RUN HOME_DIR=/root && \
     rm -rf ${HOME_DIR}/oss_compliance*
 
 ENTRYPOINT ["./entrypoint.sh"]
+
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.ml.engines.sagemaker.dlc.framework.huggingface.tgi="true"
+LABEL com.amazonaws.sagemaker.capabilities.multi-models="true"
```
Comment (lanking520) [inline huggingface/pytorch/tgi/docker/0.5.0/py3/cu118/Dockerfile.gpu:135]: MME may not be supported

File: huggingface/pytorch/tgi/docker/0.6.0/py3/cu118/Dockerfile.gpu [hunk 2]
```diff
@@ -172,3 +172,8 @@ COPY --chmod=775 sagemaker-entrypoint.sh entrypoint.sh
 RUN sed -i '7 i export MODEL_ID="${HF_MODEL_ID}"' entrypoint.sh
 
 ENTRYPOINT ["./entrypoint.sh"]
+
+LABEL dlc_major_version="1"
+LABEL com.amazonaws.ml.engines.sagemaker.dlc.framework.huggingface.tgi="true"
+LABEL com.amazonaws.sagemaker.capabilities.multi-models="true"
```
Comment (lanking520) [inline huggingface/pytorch/tgi/docker/0.6.0/py3/cu118/Dockerfile.gpu:178]: remove mme

**Review by lanking520 — CHANGES_REQUESTED** [review]

File: huggingface/pytorch/tgi/docker/0.5.0/py3/cu118/Dockerfile.gpu (hunk 1 above)
Comment (xyang16) [reply huggingface/pytorch/tgi/docker/0.5.0/py3/cu118/Dockerfile.gpu]: Done.

File: huggingface/pytorch/tgi/docker/0.6.0/py3/cu118/Dockerfile.gpu (hunk 2 above)
Comment (xyang16) [reply huggingface/pytorch/tgi/docker/0.6.0/py3/cu118/Dockerfile.gpu]: Done

**Review by lanking520 — APPROVED** [review]

**Pull request merged**